src/scielo/bin/xml/prodtools/server/mailer.py

```diff
@@ -13,6 +14,14 @@ def __init__(self, config):
                 config.email_server)
 
     def send_message(self, to, subject, text, attaches=[]):
+        if not self.config.is_enabled_email_service:
+            LOGGER.info("Could not send this email. The mailer service is disabled")
+            return None
+        elif self.mailer is None:
```

Comment:
@joffilyfe um dos problemas é a qui.
Suponha que `config.is_enabled_email_service=True`, `self.mailer is not None`, porque ele é uma instância de `email_service.EmailService`, mas não necessariamente `config.email_server` tem o serviço habilitado. No dsteste, `self.mailer` é instanciado, mas dá exceção ao enviar email porque não há o serviço de email.

Reply:
Se entendi bem @robertatakenaka pode acontecer do e-mail estar ativo mas o serviço não, certo?

Da forma que o código existe hoje nós não temos como saber se a conexão será estabelecida até que haja uma tentativa de envio das mensagens. Mesmo assim a exceção não interrompe a execução do programa.

Você acha que deveríamos mudar como a classe `EmailService` opera?